protolite-well-known-types/gradle.properties

```diff
@@ -1,5 +1,5 @@
 # IMPORTANT (b/285892320) Keep version and latestReleasedVersion in sync 
 # unless you are releasing a new version of the library to prevent issues 
 # with transitive dependencies.
-version=18.0.0
+version=18.0.1
```

Comment:
This is WAI. See the comment above `version`

Reply:
I'll pay better attention next time! Added a hardcoded exclusion to protolite-well-known-types